src/opm/input/eclipse/Schedule/Well/Well.cpp

```diff
@@ -64,6 +64,9 @@ namespace {
         if (item.get<int>(0) == 0)
             return true;
 
+        if (item.get<int>(0) < 0)
```

Comment:
This is something needs to be discussed. 

For WELOPEN and WPIMULT, the default I, J, K values are negative
For COMLUMP, the default I, J, K values are 0. 

The observation is that for WPIMULT, the following record will apply to all connections
```
WPIMULT
  'WEL' 0.9 0 0 0 /
/
```
but the behavoir shows the commercial simulator **does not** think it is defaulted. 

But the following record, 
```
WPIMULT
  'WEL' 0.9 -1 -1 -1 /
/
```
the commercial simulator treats it as **defaulted** and also apply to all the connections.